src/thermo/RedlichKwongMFTP.cpp

```diff
@@ -207,10 +207,12 @@ void RedlichKwongMFTP::compositionChanged()
 
 void RedlichKwongMFTP::getActivityConcentrations(doublereal* c) const
 {
-    getPartialMolarVolumes(m_partialMolarVolumes.data());
+    doublereal pratio = pressure()/refPressure();
+    //doublereal refP = refPressure();
+    //doublereal* a;
+    getActivityCoefficients(c);
     for (size_t k = 0; k < m_kk; k++) {
-        c[k] = moleFraction(k) / m_partialMolarVolumes[k];
-    }
+        c[k] *= refPressure()*moleFraction(k)*pratio/RT();}
```

Comment:
Since `pratio` is P/Pref, you can just write this factor as `moleFraction(k)*pressure()/RT()`.

Also, as a minor style nitpick, the closing `}` should be on the next line.